sentry_telegram/plugin.py

```diff
@@ -103,7 +117,7 @@ def build_message(self, group, event):
         }
 
     def build_url(self, project):
-        return 'https://api.telegram.org/bot%s/sendMessage' % self.get_option('api_token', project)
+        return '%s/bot%s/sendMessage' % (self.get_option('api_origin', project),self.get_option('api_token', project))
```

Comment:
It would be more PEP8 compatible with space after the comma.

```suggestion
        return '%s/bot%s/sendMessage' % (self.get_option('api_origin', project), self.get_option('api_token', project))
```